stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/channel/ChannelController.kt

```diff
@@ -272,12 +272,7 @@ public class ChannelController internal constructor(
     }
 
     internal suspend fun show(): Result<Unit> {
-        channelLogic.setHidden(false)
```

Comment:
All these side effects should be removed since we can rely on events. Receiving event guarantees that it happened on BE side. Here we did an optimistic update which can be wrong sometimes